fingerprint2.js

```diff
@@ -899,6 +870,34 @@
         return null
       }
     },
+    getWebGlShaderPrecisionFormat: function () {
```

Comment:
inline this function into the place where it's called, and get rid of the array copying. Directly push to the result array.